lib/PuppeteerSharp.Tests/InputTests/ClickTests.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PuppeteerSharp.Input;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.InputTests
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class ClickTests : PuppeteerPageBaseTest
+    {
+        private const string Dimensions = @"function dimensions() {
```

Comment:
this constant appears also in `InputTests.cs`, maybe reference the same constant from both classes?

Reply:
Good catch. Removed.